.github/workflows/ci.yaml

```diff
@@ -6,61 +6,52 @@ on:
       - 'jenkins-scripts/dsl/**'
 
 jobs:
-  dsl_ci:
+  xml_generation:
     runs-on: ubuntu-latest
-    name: Diff for DSL code
+    name: Generate XML config from DSL
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 2
       - uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
           java-version: '11'
       - name: Download and setup job dsl jar
         run: ./jenkins-scripts/dsl/tools/setup_local_generation.bash
-      - name: Generate all DSL files
+      - name: Generate XML files
         run: |
+          # Remove existing logs files. Code is adding content to them but not re-generating
           # simulate token for brew_release
           sudo mkdir -p /var/lib/jenkins/ && sudo touch /var/lib/jenkins/remote_token
           sudo chown -R ${USER} /var/lib/jenkins
-          cd jenkins-scripts/dsl
+          pushd jenkins-scripts/dsl
+          rm -r logs/*
           WRITE_JOB_LOG=1 java -jar tools/jobdsl.jar *.dsl
-      - name: Checks for DSL Code
-        run: |
-          cd jenkins-scripts/dsl
-          ./dsl_checks.bash
-      - name: Export XML generated configuration for diff
-        run: |
-          cd jenkins-scripts/dsl
-          # export files for later diff
-          mkdir /tmp/pr_xml_configuration && mkdir /tmp/pr_log_generated/
-          # sort jobs.txt
-          sort jobs.txt -o jobs.txt
-          mv *.xml /tmp/pr_xml_configuration/
-          mv *.txt /tmp/pr_log_generated/
+          find logs/* -exec sort {} -o {} \;
+          popd
+      - name: Update and commit logs generated
+        uses: stefanzweifel/git-auto-commit-action@v5
```

Comment:
By default this action adds the author whose commit triggered the action as a commit co-author. Does the configuration below preserve that extra information?

Reply:
It does not. Default behavior is nicer indeed [e973aeb](https://github.com/gazebo-tooling/release-tools/pull/1089/commits/e973aebb7923f3d5aad746928349d186314425e3)